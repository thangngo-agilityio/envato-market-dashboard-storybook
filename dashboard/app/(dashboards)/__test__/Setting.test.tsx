// Pages
import Settings from '../settings/page';

const { render } = testLibReactUtils;

describe('Settings render', () => {
  test('Should render match with snapshot.', () => {
    const { container } = render(<Settings />);

    expect(container).toMatchSnapshot();
  });
});
