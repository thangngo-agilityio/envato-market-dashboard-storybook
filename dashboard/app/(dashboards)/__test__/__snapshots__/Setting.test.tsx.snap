// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings render Should render match with snapshot. 1`] = `
<div>
  <div
    class="css-mcjfvc"
  >
    <div
      class="css-1tst7i4"
    >
      <div
        class="css-1ki54i"
      >
        <div
          class="css-1c513bj"
        >
          <div
            class="css-1eejh74"
          />
          <div
            class="css-0"
          >
            <h4
              class="chakra-heading css-9itg8i"
            >
              Personal Informations
            </h4>
            <p
              class="chakra-text css-1rwrau8"
            >
              Fill in your personal information
            </p>
          </div>
        </div>
      </div>
      <div
        class="css-1ki54i"
      >
        <div
          class="css-1kzck66"
        >
          <div
            class="css-1oje7ag"
          />
          <div
            class="css-0"
          >
            <h4
              class="chakra-heading css-9itg8i"
            >
              FAQ
            </h4>
            <p
              class="chakra-text css-1rwrau8"
            >
              Frequently asked questions
            </p>
          </div>
        </div>
      </div>
      <div
        class="css-1ki54i"
      >
        <div
          class="css-1kzck66"
        >
          <div
            class="css-1oje7ag"
          />
          <div
            class="css-0"
          >
            <h4
              class="chakra-heading css-9itg8i"
            >
              Security
            </h4>
            <p
              class="chakra-text css-1rwrau8"
            >
              Change your password
            </p>
          </div>
        </div>
      </div>
      <div
        class="css-1ki54i"
      >
        <div
          class="css-1kzck66"
        >
          <div
            class="css-1oje7ag"
          />
          <div
            class="css-0"
          >
            <h4
              class="chakra-heading css-9itg8i"
            >
              Term and Conditions
            </h4>
            <p
              class="chakra-text css-1rwrau8"
            >
              Term and Conditions of use and privacy policy
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      class="css-1d84bax"
    >
      <div
        class="css-9ebwd5"
      >
        <div
          class="chakra-spinner css-1x3wzxw"
        >
          <span
            class="css-8b45rq"
          >
            Loading...
          </span>
        </div>
      </div>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
